src/components/login/login.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@material-ui/core';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+interface State {
+  APIClient: rest.Evergreen;
+  open : boolean;
+  submitted : boolean
+  username : string
+  password : string
+}
+
+class Props {
+}
+
+export class Login extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.handleClickOpen = this.handleClickOpen.bind(this);
+    this.handleClickClose = this.handleClickClose.bind(this);
+    this.handleClickSubmit = this.handleClickSubmit.bind(this);
+    this.state = {
+      APIClient: rest.EvergreenClient("admin", "e4f2c40463dcade5248d36434cb93bac", "http://localhost:8080/api"),
+      open : false,
+      submitted : false,
+      username : "",
+      password : "",
+    };
+  }
+
+  public handleClickOpen() : any{
+    this.setState({
+      open : true,
+    });
+  }
+
+  public handleClickClose(): any {
+    this.setState({
+      open : false,
+    });
+  }
+
+  public handleClickSubmit(): any {
+    console.log(this.state.username);
+    this.setState({
+      open : false,
+      submitted : true,
+    });
+  }
+
+  public render() {
+    return (
+      <div>
+        <Button onClick={this.handleClickOpen}>
```

Comment:
I think this button belongs in the menu in App.tsx. That way the UI will be consistent and also the contents of that are in a single place (we may want to split that off into its own component later too)

Reply:
Moved to the app bar, still working on getting the positioning correct though.

Reply:
Just to clarify, I meant the dropdown menu with this comment, not the toolbar. Though if you think it makes more sense there, that works too